pkgs/by-name/un/unstoppableswap-gui/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  dpkg,
+  autoPatchelfHook,
+  cairo,
+  gdk-pixbuf,
+  webkitgtk_4_1,
+  gtk3,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  name = "unstoppableswap-gui";
+  version = "2.0.3";
+
+  src = fetchurl {
+    url = "https://github.com/UnstoppableSwap/core/releases/download/${finalAttrs.version}/UnstoppableSwap_${finalAttrs.version}_amd64.deb";
```

Comment:
This repo is renamed. Maybe this package should also be renamed?

Reply:
I'm sorry, not very familiar with the naming conventions, based on the repo name, what would you suggest the package name be?

Reply:
Oh, wow. When I opened this PR, it was still UnstoppableSwap.